File: src/SurveySays/wwwroot/ngApp/models/surveyToTake.ts
namespace SurveySays.Models {
    
    export interface ISurveyToTake {
        id: number;
        firstName: string;
        lastName: string;
        anonymous: boolean;
        dateTaken: string;
        troopNum: number;
        course: ICourse;
        survey: ISurvey;
        answers: IAnswer[];
    }

    export class SurveyToTake implements ISurveyToTake {
        id: number;
        firstName: string;
        lastName: string;
        anonymous: boolean;
        dateTaken: string;
        troopNum: number;
        course: ICourse;
        survey: ISurvey;
        answers: IAnswer[];
    }

}

File: src/Domain/Models/SurveyToTake.cs
using Domain.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain.Models
{
    public class SurveyToTake
    {
        public SurveyToTake()
        {
            Answers = new List<Answer>();
        }
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public bool Anonymous { get; set; }
        public Course Course { get; set; }
        public Survey Survey { get; set; }
        public Guid FullGuid { get; set; }
        public string ShortGuid { get; set; }
        public ICollection<Answer> Answers { get; set; }
    }
}
